src/app/admin-view/admin-forms/researcher-form/researcher-form.component.ts

```diff
@@ -45,10 +48,13 @@ export class ResearcherFormComponent implements OnInit, OnDestroy {
     if (this.researcher != null ) {
       this.title = 'Edition';
       this.fillForm();
+    } else {      
+      this.researcherForm.get('type').setValue(ResearcherTypes.Researcher);
```

Comment:
Não precisa pré-selecionar um tipo quando for insert ") esse else pode sair, o resto tá bacana;

Reply:
O branch foi deletado, devo criar um novo PR para corrigir isso?

Reply:
Precisa não, arrumei lá já, tá deletado porque já foi pra master. 😎